framework/src/modules/chain/helpers/config.js

```diff
@@ -119,7 +119,8 @@ function Config(packageJson, parseCommandLineOptions = true) {
 		},
 		redis: {
 			host: program.redis || getenv(process.env.LISK_REDIS_HOST),
-			port: parseInt(getenv(process.env.LISK_REDIS_PORT)) || null,
+			port:
+				+program.port || parseInt(getenv(process.env.LISK_REDIS_PORT)) || null,
```

Comment:
```suggestion
				parseInt(program.port || getenv(process.env.LISK_REDIS_PORT)) || null,
```
Port can only be an integer